content/guides/2.library_creation.md

```diff
@@ -7,13 +7,14 @@ fullpath: /guides/library_creation
 ---
 
 # Introduction
-This guide will help you to create a library in ABS. This guide assumes you have correctly configured your installation either in Docker or by compiling from source. Some troubleshooting tips are included at the end of this guide.
+This guide will help you to create a library in ABS. This guide assumes you have correctly configured your installation.
 
-Screenshots are from an install of v2.2.0 in Docker.
+Screenshots are from an install of v2.2.0 in Docker, but remain mostly unchanged in v2.2.20.
 
-# Setting up the library
+# Setting up a library
 
-Libraries can contain either books or podcasts. To create a library file in ABS:
+ABS supports two kinds of libraries: book and podcast. The podcast library can automatically download new episodes from an RSS feed, but this is covered in another guide [insert link].
```

Comment:
Is this a guide you are working on?

Reply:
Yes, I'm working on a podcast guide and thought it fit better there (haven't gotten very far with it yet).

Could be written as "... RSS feed, covered more in this guide (link)" once I finish the guide, and leave that last clause out for now.